src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -675,8 +675,8 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 			livingEntity.ridable = _true;
 			livingEntity.canControlStrafe = !_true;
 			livingEntity.canControlForward = _true;
-			livingEntity.canTrade = true;
-			livingEntity.villagerTradingType = true;
+			livingEntity.canTrade = false;
+			livingEntity.villagerTradingType = false;

```

Comment:
...rather than using hard-coded values instead of the method parameter (`_true`).